src/main.rs

```diff
@@ -15,10 +15,9 @@
 // You should have received a copy of the GNU General Public License
 // along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
-extern crate clap;
 extern crate crossbeam_channel as channel;
 #[macro_use]
-extern crate logger;
+extern crate cita_logger;
```

Comment:
Is it no need to `extern` crate in 2018 edition?

Reply:
Not include `macro`

Reply:
Okay! 
But I have wrote some code likes:
```rust
// declares a macro
use util::micro_service_init;

// use the macro
micro_service_init!("cita-network", "CITA:network", stdout);
```

before.

Reply:
I'm not sure. You can have a try and fix it.